### PR TITLE
yamllint formatting cleanup for role ocp4-workload-ccnrd

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp4-workload-ccnrd

### CODE REVIEW

File: ansible/roles/ocp4-workload-ccnrd/files/knative-sources.yaml [hunk 1]
```diff
@@ -801,7 +804,20 @@ data:
     #   PodIP         string  // IP of the pod hosting the revision
     # }
     #
-    logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.Response.Code}}, "responseSize": "{{.Response.Size}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
+    logging.request-log-template: >-
```
Comment (ritesh97-rh) [inline ansible/roles/ocp4-workload-ccnrd/files/knative-sources.yaml:807]: yaml to json conversion includes the newlines here?
`{
   "logging.request-log-template": "{\"httpRequest\":  {\"requestMethod\": \"{{.Request.Method}}\",\n   \"requestUrl\": \"{{js .Request.RequestURI}}\",\n   \"requestSize\": \"{{.Request.ContentLength}}\",\n   \"status\": \"{{.Response.Code}}\",\n   \"responseSize\": \"{{.Response.Size}}\",\n   \"userAgent\": \"{{js .Request.UserAgent}}\",\n   \"remoteIp\": \"{{js .Request.RemoteAddr}}\",\n   \"serverIp\": \"{{.Revision.PodIP}}\",\n   \"referer\": \"{{js .Request.Referer}}\",\n   \"latency\": \"{{.Response.Latency}}s\",\n   \"protocol\": \"{{.Request.Proto}}\"},\n   \"traceId\": \"{{index .Request.Header \"X-B3-Traceid\"}}\"}"
}` is the output I get from yaml to json.

Comment (jkupferer) [reply ansible/roles/ocp4-workload-ccnrd/files/knative-sources.yaml:807]: Odd, that surprises me too. I am not sure of a readable way of formatting this to make it readable... but we need to find one or add an exception.

File: ansible/roles/ocp4-workload-ccnrd/tasks/install-username-distribution.yaml [hunk 2]
```diff
@@ -41,14 +41,21 @@
 
 - name: construct url argument for username distribution
   set_fact:
-    guides_urls: "{{ guides_urls + ['http://guides-' + item.name + '-labs-infra.' + route_subdomain + item.path + '?userid=%USERNAME%;' + item.title ] if (item.name in modules) else guides_urls }}"
+    guides_urls: >-
+      {{ guides_urls +
+      ['http://guides-' + item.name + '-labs-infra.'
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-ccnrd/tasks/install-username-distribution.yaml:46]: Jinja2 string concatenation should use `~` operator rather than `+`

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles/ocp4-workload-ccnrd/files/knative-sources.yaml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp4-workload-ccnrd/files/knative-sources.yaml:807]: Will add an exception for now

File: ansible/roles/ocp4-workload-ccnrd/tasks/create_che_workspace.yaml [hunk 3]
```diff
@@ -15,7 +15,11 @@
 
 - name: Create workspace for {{ user }} from devfile
   uri:
-    url: "https://codeready-labs-infra.{{ route_subdomain }}/api/workspace/devfile?start-after-create=true&namespace={{ user }}"
+    url: >-
+      https://codeready-labs-infra.{{
+      route_subdomain
+      }}/api/workspace/devfile?start-after-create=true&namespace={{
+      user }}
```
Comment (jkupferer) [inline ansible/roles/ocp4-workload-ccnrd/tasks/create_che_workspace.yaml:22]: This is not resonable/readable

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Comment by jkupferer** [comment]
Conflicts...

**Pull request closed without merging**